pex/pex.py

```diff
@@ -520,3 +522,31 @@ def run(self, args=(), with_chroot=False, blocking=True, setsid=False, **kwargs)
                                     stderr=kwargs.pop('stderr', None),
                                     **kwargs)
     return process.wait() if blocking else process
+
+  def do_entry_point_verification(self):
```

Comment:
Given comments above re uniformity in performing the validation, I'd make this _private.

Reply:
Corrected.